code/modules/mob/living/carbon/alien/life.dm

```diff
@@ -1,161 +0,0 @@
-// Alien larva are quite simple.
-/mob/living/carbon/alien/Life()
-
-	set invisibility = 0
-	set background = 1
-
-	if (transforming)	return
-	if(!loc)			return
-
-	..()
-
-	if (stat != DEAD) //still breathing
-		// GROW!
-		update_progression()
-
-	blinded = null
-
-	//Status updates, death etc.
-	update_icons()
-
-/mob/living/carbon/alien/handle_mutations_and_radiation()
-
-	// Currently both Dionaea and larvae like to eat radiation, so I'm defining the
-	// rad absorbtion here. This will need to be changed if other baby aliens are added.
-
-	if(!radiation)
-		return
-
-	var/rads = radiation/25
-	radiation -= rads
-	adjust_nutrition(rads)
-	heal_overall_damage(rads,rads)
-	adjustOxyLoss(-(rads))
-	adjustToxLoss(-(rads))
-	return
-
-/mob/living/carbon/alien/handle_regular_status_updates()
-
-	if(status_flags & GODMODE)	return 0
-
-	if(stat == DEAD)
-		blinded = 1
-		silent = 0
-	else
-		updatehealth()
-		if(health <= 0)
-			death()
-			blinded = 1
-			silent = 0
-			return 1
-
-		if(paralysis && paralysis > 0)
-			blinded = 1
-			set_stat(UNCONSCIOUS)
-			if(halloss > 0)
-				adjustHalLoss(-3)
-
-		if(sleeping)
-			adjustHalLoss(-3)
-			if (mind)
-				if(mind.active && client != null)
-					AdjustSleeping(-1)
-			blinded = 1
-			set_stat(UNCONSCIOUS)
-		else if(resting)
-			if(halloss > 0)
-				adjustHalLoss(-3)
-
-		else
-			set_stat(CONSCIOUS)
-			if(halloss > 0)
-				adjustHalLoss(-1)
-
-		// Eyes and blindness.
-		if(!has_eyes())
-			SetBlinded(1)
-			blinded =    1
-			eye_blurry = 1
-		else if(eye_blind)
-			AdjustBlinded(-1)
-			blinded =    1
-		else if(eye_blurry)
-			eye_blurry = max(eye_blurry-1, 0)
-
-		update_icons()
-
-	return 1
-
-/mob/living/carbon/alien/handle_regular_hud_updates()
-
-	if (stat == 2 || (XRAY in src.mutations))

```

Comment:
Gradual removal of references to genetic mutations in hopes of someday being able to rip the system out